test/powershell/engine/Remoting/SSHRemotingAPI.Tests.ps1

```diff
@@ -26,16 +45,51 @@ Describe "SSH Remoting API Tests" -Tags "Feature" {
                 $sshConnectionInfo = [System.Management.Automation.Runspaces.SSHConnectionInfo]::new(
                     "UserName",
                     "localhost",
-                    "NoValidKeyFilePath")
+                    "NoValidKeyFilePath",
+                    22)
 
                 $rs = [runspacefactory]::CreateRunspace($sshConnectionInfo)
                 $rs.Open()
 
-                throw "No Exception!"
+                throw "SSHConnectionInfo did not throw expected FileNotFoundException exception"
```

Comment:
Please use `throw "No Exception!"` - it is our [documented template](https://github.com/PowerShell/PowerShell/blob/master/docs/testing-guidelines/WritingPesterTests.md)